main.js

```diff
@@ -34,6 +34,14 @@ const formatReplacementsMap = {
 	a: '(date.getHours() >= 12 ? "pm" : "am")' // pm
 };
 
+const inSeconds = {
+	hour: 60 * 60,
```

Comment:
Any reason for not adding `minute` to the `inSeconds` object? Seems odd to leave it out considering the rest of the changes made.